### ISSUE TITLE
OF Blacklists don't seem to work on latest commit

### ISSUE THREAD

**Issue by Jakeyy22** [DIGITALCRIMINALS/OnlyFans#824]
Updated to the latest commit, and the OF blacklist feature doesn't appear to work.

Kept pulling all subbed models down, reverted to build from the 27th Jan, for the blacklists to work again.

Tried the config file with multiple text methods no joy.

"blacklists": ["Ignore"],
or
"blacklists": "Ignore",

**Comment by DIGITALCRIMINALS** [comment]
I fixed it now, for sure.

**Comment by DIGITALCRIMINALS** [comment]
Did you run the updater.py

**Comment by Jakeyy22** [comment]
> Did you run the updater.py

I'm pretty sure I did.

I'll be able to try the latest commit again in a few hours.

**Comment by lamepennies** [comment]
Can confirm with newest update, this is still an issue.

**Comment by Jakeyy22** [comment]
Yep even tried the commit from 7 hours ago and still no blacklist working, with a updater.py run.

PS E:\OnlyFans-master> poetry run python .\updater.py
Updating dependencies
Resolving dependencies...

Writing lock file

Package operations: 0 installs, 25 updates, 0 removals

  • Updating charset-normalizer (2.1.1 -> 3.1.0)
  • Updating urllib3 (1.26.14 -> 1.26.15)
  • Updating babel (2.11.0 -> 2.12.1)
  • Updating docutils (0.17.1 -> 0.18.1)
  • Updating sphinxcontrib-htmlhelp (2.0.0 -> 2.0.1)
  • Updating typing-extensions (4.4.0 -> 4.5.0)
  • Updating wrapt (1.14.1 -> 1.15.0)
  • Updating aiohttp (3.8.3 -> 3.8.4)
  • Updating astroid (2.13.3 -> 2.15.0)
  • Updating greenlet (2.0.1 -> 2.0.2)
  • Updating mypy-extensions (0.4.3 -> 1.0.0)
  • Updating soupsieve (2.3.2.post1 -> 2.4)
  • Updating python-socks (2.1.1 -> 2.2.0)
  • Updating sphinxcontrib-jquery (2.0.0 -> 4.1)
  • Updating beautifulsoup4 (4.11.1 -> 4.11.2)
  • Updating orjson (3.8.5 -> 3.8.7)
  • Updating sqlalchemy (1.4.46 -> 2.0.6)
  • Updating sphinx-rtd-theme (1.1.1 -> 1.2.0)
  • Updating zipp (3.11.0 -> 3.15.0)
  • Updating alembic (1.9.2 -> 1.10.2)
  • Updating importlib-metadata (4.13.0 -> 6.0.0)
  • Updating tqdm (4.64.1 -> 4.65.0)
  • Updating ultima-scraper-api (0.0.1.5 -> 0.1.4)
  • Updating ultima-scraper-collection (0.1.3 -> 0.1.4)
  • Updating ultima-scraper-renamer (0.0.3 -> 0.1.3)
Script has been updated, exiting in 5 seconds
PS E:\OnlyFans-master> poetry run python .\updater.py
Updating dependencies
Resolving dependencies...

Writing lock file

No dependencies to install or update
Script has been updated, exiting in 5 seconds
PS E:\OnlyFans-master> poetry run python .\start_us.py
Choose Subscriptions: 0 = All |

**Comment by twist3dimages** [comment]
Same here

**Comment by twist3dimages** [comment]
Blacklist doesn't work for me either
